### ISSUE TITLE
Only open .zip files if file explorer is set to open them by default

### ISSUE THREAD

**Issue by josh-65** [files-community/Files#6110]
### What's the Problem?

I have 7-zip set to open all archives and right now files doesn't use that. It tries using itself

### Solution/Idea

It should only attempt to open an archive if file explorer is set to open that type by default.

### Alternatives

An alternative is allowing to set files as the default to open .zips in windows default app protocols. (Settings > Apps > default apps > Chose default applications by file type)

### Files Version

2.0.12.0

### Windows Version

10.19043.1202

**Comment by gave92** [comment]
I agree we should open it in Files if either explorer or Files is the default app for zips.
